tests/phpunit/tests/date/wpTimezone.php

```diff
@@ -31,13 +42,29 @@ public function test_should_convert_gmt_offset( $gmt_offset, $tz_name ) {
 	 * @ticket 24730
 	 */
 	public function test_should_return_timezone_string() {
-		update_option( 'timezone_string', 'Europe/Kiev' );
+		update_option( 'timezone_string', 'Europe/Helsinki' );
+
+		$this->assertSame( 'Europe/Helsinki', wp_timezone_string() );
```

Comment:
These assertions should have a `$message` parameter.

Reply:
Agreed, but improving the tests as-such is outside the scope of this ticket. This commit only changes the timezone name used, nothing else.